Automattic/jetpack
# Add vr shortcode

#### Add VR shortcode feature

#### Testing instructions:

* Add vr shortcode to post: `[vr url=path-to-photo.jpg view=360]`
* Confirm the 360 photo displays properly

See: https://en.support.wordpress.com/embedding-360-photos-and-virtual-reality-vr-content/

#### Proposed changelog entry for your changes:

* Adds VR shortcode to embed 360° photos
